nozzle/api_client.go

```diff
@@ -45,8 +46,12 @@ func newAppInfo(app cfclient.App) *AppInfo {
 
 // NewAPIClient crate a new ApiClient
 func NewAPIClient(conf *NozzleConfig) (*APIClient, error) {
+	apiURL := conf.APIURL
+	if !isValidURL(apiURL) {
+		apiURL = "https://" + apiURL
```

Comment:
check if the apiURL is valid again and throw error if still invalid?

Reply:
the api costuming it will do that for us, with a pretty clear error.